resources/languages/resources.properties

```diff
@@ -1232,6 +1232,7 @@ plugin.sipaccregwizz.PREFERRED_TRANSPORT=Preferred transport
 plugin.sipaccregwizz.ADVANCED_OPTIONS=Advanced options
 plugin.sipaccregwizz.PROXY_OPTIONS=Proxy options
 plugin.sipaccregwizz.PROXY_AUTO=Configure proxy automatically
+plugin.sipaccregwizz.PROXY_FORCE_BYPASS=Configure force proxy bypass (for HA scenarios)
```

Comment:
I'm inclined to name this something like "Bypass proxy security check for HA scenarios". The point in including the term "security" is to raise an awareness that this should not be enabled just because HA sounds cool.